src/collections/_documentation/development/sdk-dev/unified-api.md

```diff
@@ -217,6 +236,22 @@ the event.  For instance if an exception was captured the hint might carry the
 original exception object.  Not all SDKs are required to provide this.  The
 parameter however is reserved for this purpose.
 
+## Event Pipeline
+
+Any event that is captured by `capture_event` is processed in the following
+order and can be discarded at any of the stages, at which point no further
+processing happens.
+
+1. If the SDK is disabled, Sentry discards the event right away.
+2. The client does event sampling based on the configured sample rate.
+   Events will be discarded randomly, according to the sample rate.
+3. The scope is applied, using `apply_to_event`. The scope’s *event processors*
+   are invoked in order and can possibly discard the event.
+4. Sentry invokes the *before-send* hook, which can also discard the event.
+5. Sentry passes the event to the configured transport. The transport can discard
```

Comment:
Mild rephrase suggestion for clarity: "The transport can discard the event if it does not have a valid DSN; its internal queue is full; or due to rate limiting, as requested by the server."